### ISSUE TITLE
devise_token_auth blocks upgrade to Rails 5.1.0

### ISSUE THREAD

**Issue by christophermlne** [lynndylanhurley/devise_token_auth#875]
```
$ bundle update                                                                        [2.2.3]
Fetching git@bitbucket.org:xxx/xxx.git
Fetching gem metadata from https://rubygems.org/.........
Fetching version metadata from https://rubygems.org/...
Fetching dependency metadata from https://rubygems.org/..
Resolving dependencies........................................................
Bundler could not find compatible versions for gem "railties":
  In Gemfile:
    devise_token_auth (= 0.1.40) was resolved to 0.1.40, which depends on
      devise (<= 4.2, > 3.5.2) was resolved to 3.5.3, which depends on
        railties (< 5, >= 3.2.6)
```

**Comment by christophermlne** [comment]
looks like this is the issue https://github.com/plataformatec/devise/issues/4510

this library will need to be updated to depend on a newer version of devise once they release a fix

**Comment by heptaman** [comment]
Hi Team! Still no solution yet? I'm trying on MacOS but still getting the error:
```
Bundler could not find compatible versions for gem "devise":
  In Gemfile:
    devise

    devise x64-mingw32

    devise_token_auth (~> 0.1.40) x64-mingw32 was resolved to 0.1.40, which depends on
      devise (<= 4.2, > 3.5.2)

    devise_token_auth (~> 0.1.40) x64-mingw32 was resolved to 0.1.40, which depends on
      devise (<= 4.2, > 3.5.2) x64-mingw32

```
Have tried everything in SO, but no success...

**Comment by christophermlne** [comment]
@heptaman the Devise team haven't cut a new release or pushed to ruby gems yet to support Rails 5.1.0. Until that happens this gem cannot be updated to use the new Devise (I tried to get around it by forking this gem and updating the gemfile/gemspec, but it didn't work because bundler has safeguards designed to prevent gem authors from using git dependencies in production).

**Comment by mvz** [comment]
This gem can already pre-emptively allow devise 4.3.0 in its gemspec to allow downstream users to use the master branches for both projects right now, and to use the new version of devise as soon as it's released.

**Comment by mvz** [comment]
@christophermlne right now you can use this gem with Rails 5.1.0 by using this in your Gemfile:

```
gem 'rails', '~> 5.1.0'
gem 'devise', git: 'https://github.com/plataformatec/devise.git', ref: 'a62faa2c8258ba1c35fe84e147f42c35eccccb8a'
gem 'devise_token_auth', git: 'https://github.com/lynndylanhurley/devise_token_auth.git'
```

**Comment by dannytatom** [comment]
@lynndylanhurley @christophermlne devise 4.3.0 has been released w/ support for rails 5.1

**Comment by romdim** [comment]
The solution above does not work anymore, as devise 4.2.1 does not support railties 5.1.0 and devise_token_auth does not yet support devise 4.3.0 that supports railties 5.1.0.
Please release or update master whenever possible. 4.3 does only that, updates the railties as I can see from https://github.com/plataformatec/devise/blob/master/CHANGELOG.md

**Comment by mvz** [comment]
@romdim do you mean my solution in https://github.com/lynndylanhurley/devise_token_auth/issues/875#issuecomment-301010930? That should still work. What output do you get?

**Comment by romdim** [comment]
You're right @mvz, my bad, I didn't take it by reference, used the master branch instead.
Still, a release would be preferable in order not to include git in docker's dependencies.

**Comment by mvz** [comment]
> Still, a release would be preferable

Definitely!

**Comment by booleanbetrayal** [comment]
Closed via #891 -- Will try to cut builds shortly

**Comment by booleanbetrayal** [comment]
Just pushed v0.1.42
